### PR TITLE
Added support for language change

### PR DESCRIPTION
**Pull request by fedelcar**
# Summary

Language can be changed from users/edit.
# Trello card

https://trello.com/c/adW2WdJr

### CODE REVIEW

**Comment by fedelcar** [comment]
@epintos Here's the PR for language switch.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,9 +2,16 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  before_action :set_locale
- 
-  def set_locale
-    I18n.locale = I18n.default_locale
-  end
+	before_action :set_locale
```
Comment (epintos) [inline app/controllers/application_controller.rb:5]: I recommend you using `?locale` to change the language. Using cookies adds to much complexity and I'm not sure how it works with other gems

File: app/views/user_mailer/welcome_email.text.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Welcome to example.com, <%= @user.name %>
```
Comment (epintos) [inline app/views/user_mailer/welcome_email.text.erb:1]: erb :dancer:

File: test/mailers/user_mailer_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (epintos) [inline test/mailers/user_mailer_test.rb:1]: You can remove this

File: test/mailers/previews/user_mailer_preview.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
+
```
Comment (epintos) [inline test/mailers/previews/user_mailer_preview.rb:3]: And This

**Comment by epintos** [comment]
@fedelcar Some comments

**Pull request closed without merging**